packages/handbook-v1/en/Basic Types.md

```diff
@@ -175,42 +175,79 @@ let colorName: string = Color[2];
+In these cases, we want to provide a type that tells the compiler and future readers that this variable could be anything, so we give it the `unknown` type.
 
 ```ts twoslash
-let notSure: any = 4;
+let notSure: unknown = 4;
 notSure = "maybe a string instead";
 notSure = false; // okay, definitely a boolean
 ```
 
-The `any` type is a powerful way to work with existing JavaScript, allowing you to gradually opt-in and opt-out of type checking during compilation.
 You might expect `Object` to play a similar role, as it does in other languages.
-However, variables of type `Object` only allow you to assign any value to them. You can't call arbitrary methods on them, even ones that actually exist:
+However, variables of type `Object` do not allow you to assign `null` or `undefined` when `--strictNullChecks` is specified.
 
 ```ts twoslash
-// @errors: 2339
-let notSure: any = 4;
-notSure.ifItExists(); // okay, ifItExists might exist at runtime
-notSure.toFixed(); // okay, toFixed exists (but the compiler doesn't check)
+// @errors: 2322
+const notSure: Object = null; // error, type 'null' is not assignable to type 'Object'
+```
+
+If you have a variable with an unknown type, you can narrow it to something more specific by doing `typeof` checks, comparison checks, or more advanced type guards that will be discussed in a later chapter:
 
-let prettySure: Object = 4;
-prettySure.toFixed(); // Error: Property 'toFixed' doesn't exist on type 'Object'.
+```ts twoslash
+// @errors: 2322 2322 2322
+declare const maybe: unknown;
+const aNumber: number = maybe; // error, 'maybe' could be a string, object, boolean, undefined, or an array
+if (maybe === true) {
+  const aBoolean: boolean = maybe; // ok
+  const aString: string = maybe; // error, 'maybe' is definitely a boolean, so definitely NOT a string
+}
+if (typeof maybe === "string") {
+  const aString: string = maybe; // ok
+  const aBoolean: boolean = maybe; // error, 'maybe' is definitely a string, so definitely NOT a boolean
+}
+```
+
+# Any
+
+In some situations, not all type information is available or it's declaration would take an inappropriate amount of effort.
+These may occur for values from code that has been written without TypeScript or a 3rd party library.
+In these cases, we might want to opt-out of type checking.
+To do so, we label these values with the `any` type:
+
+```ts twoslash
+declare function getValue(key: string): any;
+const str: string = getValue("myString"); // ok, return value of 'getValue' is not checked
+```
+
+The `any` type is a powerful way to work with existing JavaScript, allowing you to gradually opt-in and opt-out of type checking during compilation.
+Just like `unknown`, variables of type `any` allow you to assign any value to them.
+Unlike `unknown`, TypeScript let you can call arbitrary methods on them, even ones that don't exist:
```

Comment:
This line is more meant to be a comparison against `Object` rather than `unknown`.  `unknown` is **not** assignable to any type other than `unknown`, while `any` **is** assignable to any type (type checking is disabled).  Consider removing this line all together since there are a lot more differences between `any` and `Object` than just the ability to call arbitrary methods.

Reply:
Agree, I expanded this a bit in a different direction in a paragraph after after